docs/Tutorial/getting-started.md

```diff
@@ -0,0 +1,128 @@
+# Quick guide
+
+## What you'll do
+
+- Deploy the nginx-ingress-integrator charm
+- Relate it to another charm
+- Inspect the ingress it creates.
+
+We'll then also look at changing that ingress via a juju configuration update.
+
+## Requirements
+
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed. We’ll also want to make sure the ingress add-on is enabled, which we can do by running `microk8s enable ingress`.
+
+## Deploy this charm
+
+To deploy the charm and relate it to
+[the Hello Kubecon charm](https://charmhub.io/hello-kubecon) within a Juju Kubernetes model:
+
+    juju deploy nginx-ingress-integrator
+    juju deploy hello-kubecon
+    juju relate nginx-ingress-integrator hello-kubecon
+    # If your cluster has RBAC enabled you'll be prompted to run the following:
+    juju trust nginx-ingress-integrator --scope cluster
```

Comment:
We can no longer use the hello-kubecon as an example, the hello-kubecon now uses the Traefik charm as the ingress provider.

Reply:
Should we change the revision of the charm in the tutorial (it works with >18 revisions) or pick another charm? What do you think?